spec/controllers/services_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+require 'securerandom'
+RSpec.describe ServicesController, type: :controller do
+  include AuthSupport
+
+  let(:sp_component) { create(:sp_component) }
+
+  before(:each) do
+    compmgr_stub_auth
+  end
+
+  describe "GET #index" do
+    it "returns http success" do
+      get :index, params: { sp_component_id: sp_component }
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'DELETED #destroy' do
+    it 'removes an existing service' do
+      service = create(:service, sp_component_id: sp_component)
+      expect {
+        delete :destroy, params: { id: service.id }
+      }.to change(Service, :count).by(-1)
```

Comment:
this should be more robust, relying on count feels too fragile. We should test that the service we want to delete gets actually deleted...